app/static/admin/js/pages/analytics/quick-summary.js

```diff
@@ -5,53 +5,61 @@ define(["./analytics-util"], function(util) {
     ranged: {
       donor: 0,
       donation: 0,
-      item: 0
+      item: 0,
+      value: 0
     },
     total: {
       donor: 0,
       donation: 0,
-      item: 0
+      item: 0,
+      value: 0
     },
     calculated: {
       donationPerDonor: "0",
+      itemPerDonor: "0",
       itemPerDonation: "0",
+      valuePerItem: "0",
     }
   };
 
-  function getRangedData(model, startDate, endDate, force) {
+  function getRangedData(kind, startDate, endDate, force) {
+    var curQuickSummaryData;
+    if (kind === "ranged") {
```

Comment:
I am personally of the opinion that having `kind` in a function called `getRangedData` connotates the type of data. I think it would just be better to separate these into separate functions. `getRangedData` and `getTotalData`

Reply:
Yeah that's true. But getRangedData and getTotalData would literally have same (and huge) body code. Do you still think it'd be better to have them separate?

Reply:
I do think that we should have separate functions in terms of `getRanged` and `getTotal`. If you are hesitant about having duplicate code. I would create a third function called whatever that is called using this `kind` param in them.